scripts/ci/travis_build.sh

```diff
@@ -21,17 +21,17 @@ if [[ -n ${TRAVIS_TAG} ]]; then
 
 elif [[ ${TRAVIS_PULL_REQUEST} = false ]]; then
   ARCH_NUMBER=$(arch_to_build_number ${ARCH})
-  # DATE as YYYY DDD HH MM (without spaces)
-  CUR_DATE=$(date +%Y%j%H%M)
+  # get numbers of masters commits
+  NUMBER_OF_COMMITS=$(curl -I -k "https://api.github.com/repos/opengisch/QField/commits?per_page=1&sha=${CURRENT_COMMIT}" | sed -n '/^[Ll]ink:/ s/.*"next".*page=\([0-9]*\).*"last".*/\1/p')
```

Comment:
Can you check with @suricactus that CURRENT_COMMIT plays nicely with travis and gh workflows.

Reply:
I assumed it, because it's already used for the app version string. Is there a problem with that?

Reply:
It works fine with `$CURRENT_COMMIT`, I am more concerned on the way you count the commits. Didn't look in depth, but what about something like this: 
```
git rev-list --count upstream/master
```
as suggested here  https://stackoverflow.com/a/11657647 ?

Reply:
I proposed that as I wasn't completely sure if the complete commit history is fetched or if it was a shallow fetch to save time.